app/src/main/java/org/cru/godtools/activity/BasePlatformActivity.java

```diff
@@ -242,15 +244,36 @@ private void setupNavigationDrawer() {
         }
     }
 
+    /**
+     * This method is used to update the Navigation Draws Login settings.
+     * It will only show log in information if the device is set to english
+     * and display log in links based on users status.
+     * Updated by:
+     * @author Gyasi Story
+     */
     private void updateNavigationDrawerMenu() {
-        if (mLoginItem != null) {
-            mLoginItem.setVisible(mTheKey.getDefaultSessionGuid() == null);
-        }
-        if (mSignupItem != null) {
-            mSignupItem.setVisible(mTheKey.getDefaultSessionGuid() == null);
-        }
-        if (mLogoutItem != null) {
-            mLogoutItem.setVisible(mTheKey.getDefaultSessionGuid() != null);
+        if (!ConfigurationCompat.getLocales(Resources.getSystem().getConfiguration())
+                .toLanguageTags().startsWith("en")) {  // For non English
```

Comment:
Is it possible to create a constant ArrayList of language codes that support login? I had Greg create an Array and hard code "en" in it. Then he just checked if that array contained the current device language. That allows us over time to just add more things into the array rather than adding cases to an if/else or switch statement.

I realize this may be a little different b/c you're doing an `startsWith("")` instead of an exact match, but if there's a way to pull the hard coded "en" out of here into a private constant member Array that would be nice.

Reply:
The reason I did it this way is because the call that I used returns a the language and region as a collection in a string.  So the return value will be some thing like "en-US, es-US".  The first language is the main default language and the second is the secondary language.  I did the startsWith("en") to account for all the regions that speak english ("en-uk", etc..).

I will do some research to see if there is a better way to get the default Language, this way I don't have to rely on a string comparison.